tests/lifecycle/lifeparameters/parameters.go

```diff
@@ -23,27 +23,33 @@ var (
 	LifecycleTestSuiteName  = "lifecycle"
 	SkipAllButShutdownRegex = "lifecycle-pod-high-availability lifecycle-pod-scheduling" +
```

Comment:
I think it would be better if can create 1 list with all name of test cases and then remove only relevant test case name from it.

Reply:
good idea,
I will check it after the pod-recreation issue is solved.

Reply:
as agreed, this will be done in the next PR.